pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -83,7 +83,7 @@ func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
 }
 
 func status() registry.State {
-	if runtime.GOARCH != "amd64" {
+	if runtime.GOARCH != "amd64" && runtime.GOARCH != "arm64" {
```

Comment:
lets add plain "arm" in a follow up PR

Reply:
It's TODO. For now arm32 is not supported by kicbase Dockerfile.